lib/block.ml

```diff
@@ -160,19 +161,45 @@ let rec error_padding = function
       let xs = error_padding xs in
       x :: xs
 
-let pp_errors ppf t =
+let pp_error ?syntax ?delim ppf outputs =
+  match syntax with
+  | Some Syntax.Markdown ->
+      Fmt.pf ppf "```\n```mdx-error\n%a\n"
+        Fmt.(list ~sep:(any "\n") Output.pp)
+        outputs
+  | Some Syntax.Mli | Some Syntax.Mld ->
+      Fmt.pf ppf "]%a[\n{err@mdx-error[\n%a\n]err}"
```

Comment:
I think it would be nice to check that the error does not contain the string `]err}`. It's unlikely to happen, but easy to check and avoids prematurely closing the block!

Reply:
That's a good idea, done.